app/assets/stylesheets/about.scss

```diff
@@ -0,0 +1,31 @@
+@import url("http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900,400italic");
+@import url("//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css");
+$site-white: #e1e3e3;
+$site-brown: #544d3b;
+$site-black: #262626;
+$site-tan: #a48e74;
+$site-orange: #ff9e43;
+$site-cream: #ffd37c;
+$site-light-grey: #f4f7fb;
+$site-red: #e56c69;
+
+
+.middle {
+  background-color: $site-brown;
+  color: $site-white;
+  padding: 20px;
+  margin-top: 20px
+}
+
+.bw {
+  -webkit-filter: grayscale(100%);
+  -webkit-transition: all .5s ease;
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Properties should be ordered -webkit-filter, -moz-transition, -ms-transition, -o-transition, -webkit-transition, transition<br>Avoid vendor prefixes.